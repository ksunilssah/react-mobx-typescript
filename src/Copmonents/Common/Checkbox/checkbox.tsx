import * as React from 'react';
import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';

export interface ICheckboxBasicExampleState {
  isChecked: boolean;
}

export  default class CheckBox extends React.Component<{}, ICheckboxBasicExampleState> {
  constructor(props: {}) {
    super(props);

    this._onCheckboxChange = this._onCheckboxChange.bind(this);
  }

  public render(): JSX.Element {
    return (
      <div>
        <Checkbox label="Standard checkbox" onChange={this._onCheckboxChange} />
        fasf fasdf
      </div>
    );
  }

  private _onCheckboxChange() {
    console.log('he option has been changed to ');
  }
}