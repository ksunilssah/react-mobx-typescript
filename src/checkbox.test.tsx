import React from 'react';
import { shallow } from 'enzyme';
import { CheckboxBasicExample } from './checkbox';

describe('CheckboxBasic example', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<CheckboxBasicExample></CheckboxBasicExample>);
        expect(wrapper).toMatchSnapshot();
    })
});