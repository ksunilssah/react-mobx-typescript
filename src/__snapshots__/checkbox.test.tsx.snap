// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxBasic example should render correctly 1`] = `ShallowWrapper {}`;
